hail/src/main/scala/is/hail/linalg/BlockMatrix.scala

```diff
@@ -259,15 +273,15 @@ object BlockMatrix {
   }
 
   def exportBlockMatrices(
-    fs: FS,
-    bms: IndexedSeq[BlockMatrix],
-    prefix: String,
-    overwrite: Boolean,
-    delimiter: String,
-    header: Option[String],
-    addIndex: Boolean,
-    compression: Option[String],
-    customFilenames: Option[Array[String]]): Unit = {
+                           fs: FS,
+                           bms: IndexedSeq[BlockMatrix],
+                           prefix: String,
+                           overwrite: Boolean,
+                           delimiter: String,
+                           header: Option[String],
+                           addIndex: Boolean,
+                           compression: Option[String],
+                           customFilenames: Option[Array[String]]): Unit = {
```

Comment:
this is an odd formatting change? Maybe add a newline before the `)` to get the desired formatting from IntelliJ?

Reply:
Thanks, this was an accident.